spacy_llm/tasks/lemma/task.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Callable, Dict, Generic, Iterable, List, Optional, Type
+
+import jinja2
+from spacy.language import Language
+from spacy.scorer import Scorer
+from spacy.tokens import Doc
+from spacy.training import Example
+
+from ...ty import FewshotExample, TaskResponseParser
+from ..util import ExampleType, SerializableTask
+from .util import DEFAULT_LEMMA_TEMPLATE_V1
+
+
+class LemmaTask(SerializableTask[ExampleType], Generic[ExampleType]):
+    def __init__(
+        self,
+        template: str = DEFAULT_LEMMA_TEMPLATE_V1,
```

Comment:
I think we should remove the default template here and assume it's always defined?

Reply:
We do have duplicated default values for all tasks. Duplicated as in default values are defined in the factory function _and_ the task. Should we just remove all default values from all task constructors?

Reply:
I think so? Like in spaCy's factories we also only define the defaults in the `default_config`'s, right?

Reply:
From what I've seen (e. g. [`EntityLinker`](https://github.com/explosion/spaCy/blob/master/spacy/pipeline/entity_linker.py#L168)), yes.